src/browser/components/NewTeamModal.jsx

```diff
@@ -138,6 +138,7 @@ export default class NewTeamModal extends React.Component {
         className='NewTeamModal'
         show={this.props.show}
         id='newServerModal'
+        onEntered={() => this.teamNameInputRef.focus()}
```

Comment:
Just curious how come you need this?
Doesn't `autofocus` do the job?

Reply:
@devinbinnie this is to address the issue on second initialisation that @jgilliam17 found, the popup doesn't unmount when you close it and then when you show it again this needs to be set manually :)